dotCMS/src/main/java/com/dotcms/rest/api/v1/accessibility/ACheckerHelper.java

```diff
@@ -0,0 +1,100 @@
+package com.dotcms.rest.api.v1.accessibility;
+
+import com.dotcms.enterprise.achecker.ACheckerRequest;
+import com.dotcms.enterprise.achecker.ACheckerResponse;
+import com.dotcms.enterprise.achecker.dao.GuidelinesDAO;
+import com.dotcms.enterprise.achecker.impl.ACheckerImpl;
+import com.dotcms.enterprise.achecker.model.GuideLineBean;
+import com.dotcms.enterprise.achecker.tinymce.DaoLocator;
+import com.dotcms.util.EnterpriseFeature;
+import com.dotmarketing.util.UtilMethods;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ *
+ *
+ * @author Jose Castro
+ * @since Sep 4th, 2024
+ */
+public class ACheckerHelper {
+
+    private List<GuideLineBean> accessibilityGuidelineList = null;
+
+    public static final String LANG = "lang";
+    public static final String CONTENT = "content";
+    public static final String GUIDELINES = "guidelines";
+    public static final String FRAGMENT = "fragment";
+
+    /**
+     * Private constructor
+     */
+    private ACheckerHelper() {
+        // Empty constructor
+    }
+
+    /**
+     * Provides a singleton instance of the {@link ACheckerHelper}
+     */
+    private static class SingletonHolder {
+        private static final ACheckerHelper INSTANCE = new ACheckerHelper();
+    }
+
+    /**
+     * Returns a singleton instance of this class.
+     *
+     * @return A single instance of this class.
+     */
+    public static ACheckerHelper getInstance() {
+        return ACheckerHelper.SingletonHolder.INSTANCE;
+    }
+
+    /**
+     * Returns the list of Accessibility Guidelines available in the system.
+     *
+     * @return The list of Accessibility Guidelines available in the system.
+     *
+     * @throws Exception If the guidelines can't be retrieved.
+     */
+    @EnterpriseFeature
+    public List<GuideLineBean> getAccessibilityGuidelineList() throws Exception {
```

Comment:
I think both methods access db, have to be annotated with CloseDBIfOpened

Reply:
The Accessibility Checker code uses an internal H2 database for retrieving the guidelines. The DB annotations won't work there.